src/data/markdown/docs/03 cloud/04 Integrations/05 Cloud APM/02 Azure Monitor.md

```diff
@@ -81,47 +81,124 @@ export const options = {
+| tenantId<sup>(required)</sup>          | The `tenantId` provided by service principal. The directory (tenant) ID can be extracted from Azure's app registrations.                                                                                          |
+| clientId<sup>(required)</sup>          | The `clientId` provided by service principal. The application (client) ID can be extracted from Azure's app registrations.                                                                                        |
+| clientSecret<sup>(required)</sup>      | The `clientSecret` provided by service principal. The client secret can be extracted from the certificates & secrets section of Azure's app registrations.                                                        |
+| subscriptionId<sup>(required)</sup>    | The `subscriptionId` can be viewed in the subscriptions section of Azure portal.                                                                                                                                  |
+| resourceGroupName<sup>(required)</sup> | The `resourceGroupName` can be viewed in the resource groups section of Azure portal. It should match the `subscriptionId`.                                                                                       |
+| insightsAppName<sup>(required)</sup>   | The `insightsAppName` can be viewed in the application insights section of Azure portal. It should match the `resourceGroupName`.                                                                                 |
+| azureRegion                            | The `azureRegion` you've created your Azure configurations. See the list of [supported regions](#supported-regions). Default is `eastus`.                                                                         |
+| includeDefaultMetrics                  | Whether it exports the [default APM metrics](/cloud/integrations/cloud-apm/#default-apm-metrics): `data_sent`, `data_received`, `http_req_duration`, `http_reqs`, `iterations`, and `vus`. Default is `true`.     |
+| metrics                                | List of metrics to export. <br/> For more details on how to specify metrics see below.                                                                                                                            |
+| includeTestRunId                       | Whether all the exported metrics include a `test_run_id` tag whose value is the k6 Cloud test run id. Default is `false`. <br/> Be aware that enabling this setting might increase the cost of your APM provider. |
+| resampleRate                           | Sampling period for metrics in seconds. Default is 60, as Azure Monitor aggregates metrics in 1 period.                                                                                                           |
```

Comment:
> Default is 60, as Azure Monitor aggregates metrics in 1 period. 

A little confused by this sentence. Should it say "Default is 60, as Azure Monitor aggregates metrics in periods of 1 minute." ? Otherwise, I don't see the connection between "1 period" and "60" otherwise.

Reply:
yes, lost a word there